libtelemetry/src/test/java/com/mapbox/android/telemetry/SerializerTest.java

```diff
@@ -1,26 +1,46 @@
 package com.mapbox.android.telemetry;
 
+import android.app.ActivityManager;
+import android.content.Context;
 import android.location.Location;
+import android.media.AudioManager;
+import android.telephony.TelephonyManager;
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.google.gson.JsonSerializer;
 
+import org.junit.Before;
 import org.junit.Test;
 
 import java.util.Date;
 
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class SerializerTest {
 
+  @Before
```

Comment:
In general, we should avoid inline setup 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=65
Could you extract `setupMapboxTelemetry` code into a `private` "factory" method called for example `obtainMockedContext` and use it only in the tests that actually need it?
IMO, it reflects better what's going on in the test and you don't break the [triple A](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40) "rule".